src/content/headerFooterContent.js

```
@@ -0,0 +1,41 @@
+    if (DEBUG) {
+        console.log("[HEADER_FOOTER_DEBUG]", message, ...args);
+    }
+}
+
+// Message listener for popup interactions
+chrome.runtime.onMessage.addListener(function (request, sender, sendResponse) {
+    logDebug("Message received:", request);
+
+    switch (request.greeting) {
+        case 'unloadHeader':
+            const headerCode = document.querySelector('#header_content').value;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)